source/ci_source/providers/GitHubActions.ts

```diff
@@ -133,12 +133,14 @@ import { readFileSync, existsSync } from "fs"
  *
  */
 
+const eventFilePath = process.env.GITHUB_EVENT_PATH || "/github/workflow/event.json"
+
 export class GitHubActions implements CISource {
   private event: any
 
   constructor(private readonly env: Env) {
-    if (existsSync("/github/workflow/event.json")) {
-      const event = readFileSync("/github/workflow/event.json", "utf8")
+    if (existsSync(eventFilePath)) {
```

Comment:
get the GITHUB_EVENT_PATH from env instead of get from progress. Please check the call stack in  `source/commands/utils/getRuntimeCISource.ts`

Reply:
Great catch 👍